src/components/common/Dialogs/ManageWhitelistDialog/helpers.ts

```diff
@@ -30,7 +31,12 @@ export const getManageWhitelistDialogPayload = (
   } else {
     verifiedAddresses =
       whitelistAddress !== undefined
-        ? [...new Set([...whitelistedAddresses, whitelistAddress])]
+        ? [
+            ...new Set([
+              ...whitelistedAddresses,
+              createAddress(whitelistAddress),
```

Comment:
Normalise addresses so you can't add the same address twice (but one normal and one lowercase)